Formula/libvirt.rb

```diff
@@ -42,14 +34,14 @@ def install
       --with-qemu
     ]
 
-    args << "ac_cv_path_RPCGEN=#{Formula["rpcgen"].opt_prefix}/bin/rpcgen" if build.head?
+    args << "ac_cv_path_RPCGEN=#{Formula["rpcgen"].opt_prefix}/bin/rpcgen"
 
     # Work around a gnulib issue with macOS Catalina
     args << "gl_cv_func_ftello_works=yes"
 
     mkdir "build" do
-      system "../autogen.sh" if build.head?
-      system "../configure", *args
+      system "../autogen.sh", *args if build.head?
+      system "../autogen.sh", "--no-git", *args unless build.head?
```

Comment:
Cool idea... although I'm now reworking the build.head formula options since git-head currently only supports meson... and so I'm researching how to get that working too :)

Reply:
OK, latest version has working support for head builds again using meson.  I had to add a patch to fix the module ".so" suffix issue though.

 I've been discussing a solution on the libvir-list mailing list.  They feel fixing the code to correctly use the platform suffix is best -- but I haven't had a chance to figure out how to get meson to give me the module suffix yet... (I'm new to meson :)

Reply:
OK, upstream has accepted my patch to use ".dylib" on modules.  I've updated the Formula and removed the patch.